src/App.vue

```diff
@@ -16,7 +16,7 @@
 
     <ButtonComponent display-text="Prem's Button" />
 
-    <ClockComponent></ClockComponent>
+    <ClockComponent :updateInterval="5000"></ClockComponent>
```

Comment:
Kindly change `:updateInterval` to `update-interval`, because when we decalare a prop in the chils component then we use CamelCase, but when we use it in a parent component, then we use kebab-case. Also `:` is not required here as we are not doing data binding here, instead we are directly passing a value.